src/proof/supernova.rs

```diff
@@ -24,44 +24,41 @@ use crate::circuit::MultiFrame;
 use crate::coprocessor::Coprocessor;
 
 use crate::error::ProofError;
-use crate::eval::{lang::Lang, Frame, Meta, Witness, IO};
+use crate::eval::{lang::Lang, Meta};
 use crate::field::LurkField;
 use crate::proof::nova::{CurveCycleEquipped, G1, G2};
-use crate::proof::{Provable, Prover, PublicParameters};
-use crate::ptr::Ptr;
-use crate::store::Store;
+use crate::proof::{MultiFrameTrait, Provable, Prover, PublicParameters};
 
 use super::nova::NovaCircuitShape;
 
-/// Type alias for a NIVCStep with S1 field elements.
+/// Type alias for a MultiFrame with S1 field elements.
 /// This uses the <<F as CurveCycleEquipped>::G1 as Group>::Scalar type for the G1 scalar field elements
 /// to reflect it this should not be used outside the Nova context
-pub type C1<'a, F, C> = NIVCStep<'a, F, C>;
+pub type C1<'a, F, C> = MultiFrame<'a, F, C>;
```

Comment:
If you're going to make this file generic in the use of a `MultiFrame`, I would remove this alias, so as to not have it mistakenly used anywhere.